x/pki/x509/x509.go

```diff
@@ -55,6 +57,41 @@ func DecodeX509Certificate(pemCertificate string) (*X509Certificate, error) {
 	return &certificate, nil
 }
 
+func TranslateToVidOrPid(subject []byte) string {
```

Comment:
This function seems like to contain duplication. Consider refactoring it to be more generic like:
```
FormatOID(prefix string, key string)
```
and calling it separately for `vid` and `pid`